misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/MapToContainInOrderOnlyKeyValueExpectationsSpec.kt

```diff
@@ -257,6 +256,87 @@ abstract class MapToContainInOrderOnlyKeyValueExpectationsSpec(
 
         context("report options") {
             //TODO #1129 add tests, see IterableToContainInOrderOnlyValuesExpectationsSpec -> report options
+            context("") {
+                it("shows only failing with report option `showOnlyFailing`") {
+                    expect {
+                        expect(map).toContainFun(
+                            keyValue("a") { toEqual(1) },
+                            keyValue("b") { toEqual(2) },
+                            keyValue("c") { toEqual(3) },
+                            report = { showOnlyFailing() }
+                        )
+                    }.toThrow<AssertionError> {
+                        message {
+                            toContainSize(2, 3)
+                            elementSuccess(0, "1", "a", "1")
```

Comment:
> shows only failing with report option

You are not expecting the contrary here. You should check that successful elements are not in the error report

Reply:
Hi, @robstoll 

Thank you for reviewing. Asking questions you once before, how can I check that successful elements are not in the error report? I could not find the way to check specs I wrote...

Reply:
Sorry, did not get that this question was not solved yet. I remember that you were asking whether you can copy methods from IterableToContainsSepcBase to MapLikeToContainSpecBase. So yes, you totally can. You can for instance copy notToContainElement from there and adopt to map. I suggest we name the method `notToContainEntry` and it would suffice if it takes the key i.e it would look like follows:  
```
fun Expect<String>.notToContainEntry(key: String): Expect<String> = ...
```
I left out the body on purpose, let me know if you don't want to figure this out yourself, I can also provide it.

Reply:
>  I suggest we name the method notToContainEntry and it would suffice if it takes the key i.e it would look like follows:

Thanks to provide some example.  I understood that I could write expectation by `toContainFun` , but sould I create something like `notToContainEntry` ?

Reply:
Yes, you should create notToContainEntry in MapLikeToContainSpecBase